### PR TITLE
chore: deploy spore contract from reproducible environment

### PR DESCRIPTION
**Pull request by ashuralyk**
# Description

since we encountered reproducible problem while the first mainnet deployment process, which conducted different `code_hash` of contract Spore in different compiling environment, even based on a total same docker image.

however, moving reproducible process from native to CI workflow could solve problem, see [here](https://github.com/sporeprotocol/spore-contract/blob/master/.github/workflows/capsule.yaml#L59).

### CODE REVIEW

**Comment by ashuralyk** [comment]
@Flouse the previous deployed Spore contract is still on there, I can consume it later to recycle locked CKB for our later deployment use, what do you think?

**Comment by Flouse** [comment]
> the previous deployed Spore contract is still on there, I can consume it later to recycle locked CKB for our later deployment use, what do you think?

yes.
I think you could leave it there as long as you have enough CKB for new deployments.

File: deployment/migration/mainnet/spore/2024-01-25-0.2.2-beta.1.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "cell_recipes": [
+    {
+      "name": "spore",
+      "tx_hash": "0x96b198fb5ddbd1eed57ed667068f1f1e55d07907b4c0dbd38675a69ea1b69824",
+      "index": 0,
+      "occupied_capacity": 7681300000000,
+      "data_hash": "0x4a4dce1df3dffff7f8b2cd7dff7303df3b6150c9788cb75dcf6747247132b9f5",
```
Comment (Flouse) [inline deployment/migration/mainnet/spore/2024-01-25-0.2.2-beta.1.json:8]: checked

Comment (Flouse) [reply deployment/migration/mainnet/spore/2024-01-25-0.2.2-beta.1.json:8]: Consistent with https://github.com/sporeprotocol/spore-contract/actions/runs/7650122822/job/20845772777?pr=55#step:5:12

**Review by Flouse — APPROVED** [review]

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**